qemu/tests/cfg/slof_boot.cfg

```diff
@@ -0,0 +1,12 @@
+- slof_boot: install setup image_copy unattended_install.cdrom
+    type = boot
+    restart_vm = yes
+    kill_vm_on_error = yes
+    login_timeout = 240
```

Comment:
In my all experiments, the overall two cases could finish within short time (1~2 min). So is there other concern to increase the login_timeout?